build.gradle

```diff
@@ -258,6 +265,16 @@ project(':iceberg-api') {
 
   tasks.processTestResources.dependsOn rootProject.tasks.buildInfo
 
+  // Workaround to prevent:
+  // Task ':iceberg-api:processTestResources' uses this output of task
+  // ':spotlessInternalRegisterDependencies' without declaring an explicit or implicit dependency.
```

Comment:
do you think this dependency exists because of `tasks.processTestResources.dependsOn rootProject.tasks.buildInfo`?

Reply:
Maybe. Cross-project task dependencies are never a good idea.